orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/StageData.groovy

```diff
@@ -113,6 +113,7 @@ class StageData {
     String serverGroupName
     Boolean useSourceCapacity
     Boolean preferSourceCapacity
+    String targetCluster
```

Comment:
I'd suggest `clusterName` vs `targetCluster` to fit with the existing convention of `serverGroupName`. 

I'm not sure that the `target` prefix adds particular meaning?